app/src/main/java/org/p2p/wallet/swap/ui/orca/OrcaSwapPresenter.kt

```diff
@@ -282,6 +282,15 @@ class OrcaSwapPresenter(
                 destinationAmountUsd = destinationAmountUsd.toString()
             )
             swapAnalytics.logSwapVerificationInvoked(AuthAnalytics.AuthType.BIOMETRIC)
+            swapAnalytics.logSwapUserConfirmed(
```

Comment:
We already have event for this above. 
Maybe we should remove one of them?

Reply:
i'll ask the analyst